### PR TITLE
Delete model tff110

### PR DESCRIPTION
**Pull request by gearedmonkey**
### Summary

Deleting a model now deletes the items from the students store

### Checklist

- [x] Have you added unit and functional tests as appropriate?
- [x] Did you update/add documentation for new methods or changed functionality?
- [x] Have you merged the target branch down into this one?

### How to Review

1. View code changes
2. Run the server on the `delete-model-tff110` branch
3. Run the client on the `delete-model-tff110` branch
4. Delete models!

### Links

- [TFF-110](https://msoese.atlassian.net/browse/TFF-110)
- TheFourFifths/consus-client/pull/60

### CODE REVIEW

**Comment by gearedmonkey** [comment]
@Cafferoq api tests are not present in DEV therefore they are not present in this PR

**Comment by gearedmonkey** [comment]
@Cafferoq double comment sorry, but this PR was just a quick update for deleting the students items for that model. The previous PR for deleteModel is here with the tests you asked for https://github.com/TheFourFifths/consus/pull/33/files

File: docs/REST.md [hunk 1]
```diff
@@ -175,18 +175,15 @@ An array containing each model and its data.
 }
 ```
 
-## DELETE `/api/model`
-
```
Comment (thofmann) [inline docs/REST.md]: Nit: Why remove the newline?

File: docs/REST.md [hunk 2]
```diff
@@ -175,18 +175,15 @@ An array containing each model and its data.
...
+## DELETE `/api/model/`
```
Comment (thofmann) [inline docs/REST.md]: Nit: Why add the trailing slash?

File: docs/REST.md [hunk 3]
```diff
@@ -175,18 +175,15 @@ An array containing each model and its data.
...
 Delete a model
 
 ### Parameters
 
 * `modelAddress`: Address of the model to delete
 
 ### Sample Response
-
```
Comment (thofmann) [inline docs/REST.md]: Nit: Why remove the newline?

File: docs/REST.md [hunk 4]
```diff
@@ -175,18 +175,15 @@ An array containing each model and its data.
...
 The model that was just deleted
-
```
Comment (thofmann) [inline docs/REST.md]: Nit: Why remove the newline?

File: src/store/model-store.js [hunk 5]
```diff
@@ -51,7 +51,7 @@ class ModelStore extends Store {
         return modelsByActionId[actionId];
     }
 
-    getDeletedModel() {
+    getDeletedModel(){
```
Comment (thofmann) [inline src/store/model-store.js]: Nit: Why?

**Comment by gearedmonkey** [comment]
@thofmann run client on delete-model-tff110

**Pull request merged**